content/slides/geospatial-concepts-and-terms.qmd

```diff
@@ -279,19 +293,14 @@ Extremely concise, EPSG maintains a registry of codes that map to CRS definition
 Note there are other standard representations as well, e.g., Geography Markup Language (GML). The three listed here (WKT, Proj, EPSG codes) are some of the most common.
 :::
 
-
 ## Metadata standards
 
-* [Unified Metadata Model](https://www.earthdata.nasa.gov/unified-metadata-model-umm)
-* [NetCDF Climate and Forecast (CF) Metadata Conventions](https://cfconventions.org/)
-* [Federal Geographic Data Committee (FGDC) Content Standard for Digital Geospatial Metadata](https://www.fgdc.gov/standards/projects/FGDC-standards-projects/metadata/base-metadata/v2_0698.pdf)
-  * CF conventions use the FGDC standards as a guide.
-* [Cooperative Ocean/Atmosphere Research Data Service (COARDS) NetCDF Conventions](https://ferret.pmel.noaa.gov/Ferret/documentation/coards-netcdf-conventions)
-  * CF "[conventions generalize and extend the COARDS conventions](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/build/ch01s04.html)"
-* [ISO 19111:2019, Geographic information — Referencing by coordinates](https://www.iso.org/standard/74039.html)
-* [Open Geospatial Consortium (OGC) standards](https://www.ogc.org/standards/) 🌎
+When a dataset follows of one or more common standards, GIS practitioners and
+tools can accurately parse information about CRS, date/time, etc.
```

Comment:
Might be worth adding a slide or two about other metadata and standard ways of encoding those values as well? E.g., mention that a time dimension may need metadata about how time is measured/what calendar is used.

Reply:
Maybe just a passing mention in speaker notes? This isn't a subject I think we want to spend too much time on. I think we also had another slide that mentioned granularity somewhere...